x-pack/plugins/apm/e2e/run-e2e.sh

```diff
@@ -105,26 +105,38 @@ node ingest-data/replay.js --server-url http://localhost:$APM_SERVER_PORT --even
 # Stop if not all events were ingested correctly
 if [ $? -ne 0 ]; then
     printf "\n⚠️  Not all events were ingested correctly. This might affect test tests. \n"
+
+    # stop docker containers
+    cd ${APM_IT_DIR} && ./scripts/compose.py stop > /dev/null && cd -
+
+    printf "Aborting. Please try again."
     exit 1
 fi
 
+# create empty snapshot file if it doesn't exist
+SNAPSHOTS_FILE=cypress/integration/snapshots.js
+if [ ! -f ${SNAPSHOTS_FILE} ]; then
+    echo "{}" > ${SNAPSHOTS_FILE}
+fi
+
 #
 # Wait for Kibana to start
 ##################################################
-echo "\n${bold}Waiting for Kibana to start...${normal}"
-echo "Note: you need to start Kibana manually. Find the instructions at the top."
+printf "\n\n${bold}Waiting for Kibana to start...${normal}"
```

Comment:
`echo` prints a newline automatically and `printf` doesn't, so won't these be squished together? I generally only use `printf` if I need to use format strings like `%s`.

Reply:
With `echo` I couldn't add additional newlines since `\n` was outputted literally.

If you prefer I can switch back to `echo` and then do:

```bash
echo "" # newline
echo "${bold}Waiting for Kibana to start...${normal}"
```

Reply:
have you tried `echo -e` where you want to print a newline?

Reply:
I did not. Looking at the docs it sounds like that would work


![image](https://user-images.githubusercontent.com/209966/83258592-cfc2ee80-a1b6-11ea-8841-ebf3b02853db.png)


👍